src/data/load_geometry.ts

```diff
@@ -19,7 +19,8 @@ const MIN = -MAX - 1;
  * @param feature - the vector tile feature to load
  */
 export function loadGeometry(feature: VectorTileFeature): Array<Array<Point>> {
-    const scale = EXTENT / feature.extent;
```

Comment:
No longer needed once you can upgrade to https://github.com/maplibre/maplibre-tile-spec/pull/172/commits/4fb7a3cb3b96f503cc04e4a6e61ca41bce993295